Export/Scripts/pantheons.lua

```diff
@@ -0,0 +1,62 @@
+--
+-- export The Pantheon data
+--
+
+local function zip(a, b)
+    local zipped = { }
+	for i, _ in pairs(a) do
+		table.insert(zipped, { a[i], b[i] })
+    end
+    return zipped
+end
+
+if not loadStatFile then
+	dofile("statdesc.lua")
+end
+loadStatFile("stat_descriptions.txt")
+
+local out = io.open("../Data/3_0/Pantheons.lua", "w")
+out:write('-- This file is automatically generated, do not edit!\n')
+out:write('-- The Pantheon data (c) Grinding Gear Games\n\n')
+out:write('return {\n')
+
+for i, p in pairs(dat"PantheonPanelLayout":GetRowList("IsDisabled", false)) do
```

Comment:
Missing parentheses around `dat`. Also `i` is never used/name-shadowed by the inner loop. Please change throwaway variables to `_` instead.